src/chrome/content/rules/Linode.xml

```diff
@@ -1,32 +1,67 @@
 <!--
-	Problematic hosts in *linode.com:
+	Nonfunctional hosts in *linode.com:
+
+		- speedtest.atlanta ¹
+		- speedtest.dallas ¹
+		- speedtest.frankfurt ¹
+		- speedtest.fremont ¹
+		- speedtest.london ¹
+		- speedtest.newark ¹
+		- speedtest.singapore ¹
+		- speedtest.tokyo ¹
 
-		- status *
+		- mirrors ¹
+		- atlanta.mirrors ¹
+		- dallas.mirrors ¹
+		- frankfurt.mirrors ¹
+		- fremont.mirrors ¹
+		- london.mirrors ¹
+		- newark.mirrors ¹
+		- singapore.mirrors ¹
+		- tokyo.mirrors ¹
 
-	* StatusPage.io
+		- *.members ²
+		- *.nodebalancer ²
 
+	¹ Refused
+	² Customer load balancers and VPSes. Mismatched, refused or time out.
 
-	Fully covered hosts in *linode.com:
+	Problematic hosts in *linode.com:
 
-		- (www.)?
-		- blog
-		- forum
-		- library
-		- manager
-		- stats
-		- status	(→ linode.statuspage.io)
+		- apt ¹
+		- status ²
 
+	¹ "gnutls_handshake() warning: The server name sent was not recognized"
+	  SSL Labs says "Incorrect SNI alerts: apt.linode.com"
+	  It works in Chrome and Firefox, though.
```

Comment:
@J0WI Yes and no. If you scroll down to the bottom of the [SSL Labs report](https://www.ssllabs.com/ssltest/analyze.html?d=apt.linode.com&s=97.107.139.216), you can see there **is** a genuine issue ("Incorrect SNI alerts"). (Even if i don't know what it means.) Chrome and Firefox ignore the problem, while GnuTLS treats it as fatal, but the test isn't _wrong_.

It's a super-unimportant host (an apt repository, normally not accessed in a browser at all), so i'm inclined to disable the host and merge this as is. Maybe file a bug to follow up in the future and see if the tests can be changed to ignore it.